docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -176,325 +115,451 @@
1110110110': 1, '11111111111111111111010111100110111': 1, '11111111111111111111011001011001101': 1, '11111111111111111111011001111101110': 1, '11111111111111111111011011100111110': 1, '11111111111111111111011011111111111': 4, '11111111111111111111011100011110111': 1, '11111111111111111111011100111101110': 1, '11111111111111111111011101001111111': 1, '11111111111111111111011110011100111': 1, '11111111111111111111011110100111101': 1, '11111111111111111111011111101111100': 1, '11111111111111111111011111110101111': 1, '11111111111111111111011111111011111': 1, '11111111111111111111011111111101111': 1, '11111111111111111111011111111111110': 3, '11111111111111111111100011010011000': 1, '11111111111111111111110000001111110': 1, '11111111111111111111110101101111100': 1, '11111111111111111111110101111111111': 1, '11111111111111111111110110100111111': 1, '11111111111111111111110110111111111': 1, '11111111111111111111110111010111110': 1, '11111111111111111111110111011111011': 1, '11111111111111111111110111110111101': 1, '11111111111111111111110111110111110': 1, '11111111111111111111111000101111101': 1, '11111111111111111111111001011101100': 1, '11111111111111111111111001111101100': 1, '11111111111111111111111010111011111': 1, '11111111111111111111111010111110101': 1, '11111111111111111111111010111111110': 1, '11111111111111111111111011011111111': 1, '11111111111111111111111011100111111': 1, '11111111111111111111111011110111110': 1, '11111111111111111111111100010101011': 1, '11111111111111111111111100011100101': 1, '11111111111111111111111100011111100': 1, '11111111111111111111111100011111110': 1, '11111111111111111111111101010111110': 1, '11111111111111111111111101011111111': 4, '11111111111111111111111101101011110': 1, '11111111111111111111111101110111100': 1, '11111111111111111111111101111100101': 1, '11111111111111111111111101111111011': 3, '11111111111111111111111110000111100': 1, '11111111111111111111111110011111111': 1, '11111111111111111111111110100101011': 1, '11111111111111111111111110100111110': 1, '11111111111111111111111110101101111': 1, '11111111111111111111111110110110110': 1, '11111111111111111111111110111011110': 1, '11111111111111111111111110111011111': 1, '11111111111111111111111110111111010': 1, '11111111111111111111111110111111101': 6, '11111111111111111111111110111111111': 2, '11111111111111111111111111001110110': 1, '11111111111111111111111111011111101': 1, '11111111111111111111111111011111110': 1, '11111111111111111111111111100011110': 1, '11111111111111111111111111101100110': 1, '11111111111111111111111111101111101': 1, '11111111111111111111111111110000100': 1, '11111111111111111111111111110101110': 1, '11111111111111111111111111110111101': 1, '11111111111111111111111111110111110': 3, '11111111111111111111111111110111111': 3, '11111111111111111111111111111110101': 1, '11111111111111111111111111111110110': 1, '11111111111111111111111111111111100': 1, '11111111111111111111111111111111110': 1, '11111111111111111111111111111111111': 2}\n"
```

Comment:
I wonder if it makes sense to print this out or just plot a histogram instead

Reply:
I was just following the examples [here](https://docs.quantum.ibm.com/guides/primitives-examples#run-a-single-experiment-1), but I can plot them if that's allowed.